custom_components/healthchecksio/config_flow.py

```
@@ -165,16 +162,16 @@ async def async_step_user(
             }
         )
 
-        if user_input is not None and user_input.get(CONF_PING_ID) is not None:
+        if user_input is not None and user_input.get(CONF_PING_UUID) is not None:
             DATA_SCHEMA = DATA_SCHEMA.extend(
```

Pylint (3.3.7)

[convention] Variable name "DATA_SCHEMA" doesn't conform to snake_case naming style

(C0103)